dcc-storage-client/src/main/java/org/icgc/dcc/storage/client/download/DownloadStateStore.java

```diff
@@ -107,11 +108,12 @@ public boolean hasStarted(File stateDir, String objectId) {
   public List<Part> getProgress(File stateDir, String objectId) throws IOException {
     log.debug("Loading local progress for {} from {}", objectId, stateDir.toString());
     ObjectSpecification spec = loadSpecification(stateDir, objectId);
-    log.debug("Completed loading local object specification");
+    log.debug("Completed loading local object specification (meta file)");
```

Comment:
Might as well throw in `File stateDir, String objectId` to the message for good measure.